lib/babe/babe.go

```diff
@@ -322,12 +289,12 @@ func (b *Service) getAuthorityIndex(Authorities []types.Authority) (uint32, erro
 }
 
 func (b *Service) getSlotDuration() time.Duration {
```

Comment:
I noticed this function is not being unused, could you remove it?

Reply:
done